Curs04/src/views/new-evaluation.js

```diff
@@ -1,5 +1,328 @@
+                {
+                    legendSelectorContent,
+                    selectName : 'browser',
+                    labelText : 'Browser Engines'
+                }
+                    
+            ],
+            legendHeader : 'Javascript'
+        },
+        {
+            liContents:[
+                {
+                    legendSelectorContent,
+                    selectName : 'backend',
+                    labelText : `Backend frameworks`
+                },
+                {
+                    legendSelectorContent,
+                    selectName : 'unittesting',
+                    labelText : 'Unit Testing'
+                },
+                {
+                    legendSelectorContent,
+                    selectName : 'templatingoptions',
+                    labelText : 'Templating'
+                },
+                {
+                    legendSelectorContent,
+                    selectName : 'domhandle',
+                    labelText : 'DOM Manipulation'
+                }
+                    
+            ],
+            legendHeader : 'NodeJS'
+        },
+        
+        ]
+    
+        const footerAreaSection = {
+            footerText: '<p class="footer-description">Copyright@Softvision 2017</p>'
+        }
+    
+        const submitButton = {
+            submitLink: `
+            <div class="submit-button-section">
+                <input class="submit-button" type="submit" value="SUBMIT">
+            </div>`
+        }
+    
+        const textAreaContent = [
+            {
+                textAreaHeader: 'Should the candidate be hired?',
+                textAreaPlaceHolder: `The type of project that is suitable for the candidate
+                Is guidance required for the candidate`
+            },
+            {
+                textAreaHeader: 'General Impression',
+                textAreaPlaceHolder: `*required`
+            },
+            {
+                textAreaHeader: 'Workflow, Leadership & Soft Skills',
+                textAreaPlaceHolder: `Describe the environment in which the candidate works.
+                Describe any guidance or management experience`
+            }
+        ]
+    
+        const technicalLevelContent = { 
+            technicalHeader: "Technical level",
+            technicalLevel: ['Trainee', 'Junior', 'Middle', 'Senior'],
+            technicalLevelSection: ['1', '3', '3', '3']
+         }
+        
+         const navigationLinks = ['Evaluations', 'New Evaluation', 'Logout']
+         
+         const detailsForm = [
+             '<input class="candidate_input" type="text" name="candidate" placeholder="Candidate"/>',
+             '<input class="interviewer_input" type="text" name="interviewer" placeholder="Interviewer"/>',
+             '<input class="calendar_input" type="password" name="date" placeholder="4/22/2012"/>',
+             `<a href="evaluation.html"> <img style="width: 35px; margin-top: 18px;" src="img/2019-200.png"></a>`
```

Comment:
the style should be in less files not inline

Reply:
👍